breadcord/core_modules/module_manager/__init__.py

```diff
@@ -31,7 +29,7 @@ async def autocomplete(self, interaction: discord.Interaction, value: str, /) ->
             )
             for module in breadcord.helpers.search_for(
                 query=value,
-                objects=list(interaction.client.modules),
+                objects=tuple(interaction.client.modules),
```

Comment:
Why this change, is it minorly more performant?

Reply:
Yes, but in reality I think one could just remove the conversaion altogether